SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -798,6 +809,12 @@ void ActorSystem::DestroySubObject(const FUnrealObjectRef& ObjectRef, UObject& O
 	}
 }
 
+void ActorSystem::MarkClientInterestDirty(Worker_EntityId EntityId, bool bOverwrite)
+{
+	bool& bMapOverwrite = ClientInterestDirty.FindOrAdd(EntityId);
+	bMapOverwrite |= bOverwrite;
```

Comment:
This can just be a 1 liner right?
```suggestion
	ClientInterestDirty.FindOrAdd(EntityId) |= bOverwrite;
```

Reply:
Think the previous incantation is slightly easier to read - but not fussed